crates/rerun/src/run.rs

```diff
@@ -12,37 +12,56 @@ use re_smart_channel::{ReceiveSet, Receiver, SmartMessagePayload};
 use re_sdk::web_viewer::host_web_viewer;
 #[cfg(feature = "web_viewer")]
 use re_web_viewer_server::WebViewerServerPort;
-#[cfg(feature = "web_viewer")]
+#[cfg(feature = "server")]
 use re_ws_comms::RerunServerPort;
 
-// Note the extra blank lines between the point-lists below: it is required by `clap`.
+const SHORT_ABOUT: &str = "The Rerun Viewer and Server";
+
+// Place the important help _last_, to make it most visible in the terminal.
+const EXAMPLES: &str = r#"
+Environment variables:
+    RERUN_SHADER_PATH         The search path for shader/shader-imports. WARNING: Shaders are embedded in some build configurations.
```

Comment:
I would double down on this, this is not usable for 99% of users:
```suggestion
    RERUN_SHADER_PATH         The search path for shader/shader-imports. Only available in developer builds.